tracer/test/snapshots/AspNetMvc4Tests.Integrated.WithExpansion.__path=_Home_statusCode=200.verified.txt

```diff
@@ -42,7 +42,8 @@
       network.client.ip: ::1,
       runtime-id: Guid_1,
       span.kind: server,
-      version: 1.0.0
+      version: 1.0.0,
+      _dd.p.dm: -1
```

Comment:
Interesting that _this_ number is encoded as a string in tags, as opposed to all the other numbers which are encoded as numbers in Metrics 🤷‍♂️

Reply:
But it's not a number it's a dash and a number :)

Reply:
Yeah. Long story, but this feature went through several iterations. The next to last was `{service_name_hash}-{sampling_mechanism}`, then service name hash was removed, _but not the dash_, so now it's `{-mechanism}`. Happy to talk about it more over a beer or two.

Reply:
> Happy to talk about it more over a beer or two

![image](https://user-images.githubusercontent.com/18755388/184087932-b2c395c5-1461-4fb3-bf0a-43cf9ee95226.png)